samples/apps/copilot-chat-app/webapp/tests/chat.test.ts

```diff
@@ -1,51 +1,239 @@
+/* eslint-disable testing-library/prefer-screen-queries */
 import { expect, test } from '@playwright/test';
+import * as util from './testUtilCommon'
 
 /*
-This test is a simple end-to-end test that verifies the app can be
-loaded, and signed in. The user can send a message to the bot, and expect
-a response.
+This is a test suite to test the functionality of the copilot sample app.
+The Test suite goes through a bunch of tests, opening a new chat session
+for each test so that chat history is not polluted and the LLM isn't confused; 
+Here is a list of things we test right now, either standalone or as part of another test:
+- Server is running
+- User login
+- Page has correct title
+- Send a message to the bot, and expect a response
+- Chat History check
+- Start New Chat
+- SK Skill test: joke and fun fact
+- Change chat title
+- Document upload
+- Plan generation and execution for the Klarna plugin
+- Plan generation and execution for the Jira plugin 
+- Plan generation and execution for the Github plugin 
```

Comment:
as the suite continues to grow larger this list may become difficult to maintain.
instead of adding onto it each time, I think we could aim to have good readability throughout the tests themselves so that this list isn't needed? what do you think?